### PR TITLE
feat(arm): AppServicePHPVersion

### PR DESCRIPTION
**Pull request by shoshiGit**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

[//]: # "
    # PR Title
    Be aware that we use the title to create changelog automatically and therefore only allow specific prefixes
    - break:    to indicate a breaking change, this supersedes any of the types
    - feat:     to indicate new features or checks
    - fix:      to indicate a bugfix or handling of edge cases of existing checks
    - docs:     to indicate an update to our documentation
    - chore:    to indicate adjustments to workflow files or dependency updates
    - platform: to indicate a change needed for the platform
    Additionally a scope is needs to be added to the prefix, which indicates the targeted framework, in doubt choose 'general'.
    #    
    Allowed prefixs:
    ansible|argo|arm|azure|bicep|bitbucket|circleci|cloudformation|dockerfile|github|gha|gitlab|helm|kubernetes|kustomize|openapi|sast|sca|secrets|serverless|terraform|general|graph|terraform_plan|terraform_json
    #
    ex.
    feat(terraform): add CKV_AWS_123 to ensure that VPC Endpoint Service is configured for Manual Acceptance
"

## Description

*We converted the check AppServicePHPVersion from TERRAFORM language to the ARM language so that it also works on resources that are defined in the ARM language..*

Fixes # (issue)

### Description
*Ensure that 'PHP version' is the latest, if used to run the web app*

### Fix
*To fix the issue in code, update the php_version property in the Terraform configuration to "7.4"*

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: checkov/arm/checks/resource/AppServicePHPVersion.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        supported_resources = ['Microsoft.Web/sites']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources,
+                         missing_block_result=CheckResult.UNKNOWN)
+
+    def get_inspected_key(self) -> str:
+        return "properties/siteConfig/phpVersion"
+
+    def get_expected_value(self) -> Any:
+        return '7.4'
```
Comment (ChanochShayner) [inline checkov/arm/checks/resource/AppServicePHPVersion.py:19]: I think that the latest version now is `8.1`
Am I right? @tsmithv11

Comment (tsmithv11) [reply checkov/arm/checks/resource/AppServicePHPVersion.py:19]: Yeah, both 8.1 and 8.2 should be acceptable.

Comment (ChanochShayner) [reply checkov/arm/checks/resource/AppServicePHPVersion.py:19]: @shoshiGit You can take a reference from here - 
https://github.com/bridgecrewio/checkov/blob/main/checkov/arm/checks/resource/AppServicePythonVersion.py

File: checkov/arm/checks/resource/AppServicePHPVersion.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        supported_resources = ['Microsoft.Web/sites']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources,
+                         missing_block_result=CheckResult.UNKNOWN)
+
+    def get_inspected_key(self) -> str:
+        return "properties/siteConfig/phpVersion"
+
+    def get_expected_value(self) -> Any:
+        return '8.1' or '8.2'
```
Comment (ChanochShayner) [inline checkov/arm/checks/resource/AppServicePHPVersion.py:19]: Now we are returnoing bool from this func, and not the expected version.
Please implement it as [here](https://github.com/bridgecrewio/checkov/blob/main/checkov/arm/checks/resource/AppServicePythonVersion.py) with get_expected_values

**Review by ChanochShayner — APPROVED** [review]

**Review by achiar99 — APPROVED** [review]

**Pull request merged**